scipy/stats/unuran/license.txt

```diff
@@ -0,0 +1,357 @@
+    unuran-1.8.1/examples/example_vemp.c
+    unuran-1.8.1/experiments/Makefile.am
+    unuran-1.8.1/experiments/Makefile.in
+    unuran-1.8.1/experiments/make_timing_table.c
+    unuran-1.8.1/experiments/timings_chap_10.conf
+    unuran-1.8.1/experiments/timings_chap_8.conf
+    unuran-1.8.1/scripts/Makefile.am
+    unuran-1.8.1/scripts/Makefile.in
+    unuran-1.8.1/scripts/compute_machine_constants.c
+    unuran-1.8.1/scripts/make_urng_header.pl
+    unuran-1.8.1/scripts/merge_h.pl
+    unuran-1.8.1/scripts/read_PDF.pl
+    unuran-1.8.1/scripts/remove_comments.pl
+    unuran-1.8.1/scripts/run_valgrind.sh
+    unuran-1.8.1/scripts/win32/Makefile.win32
+    unuran-1.8.1/scripts/win32/build.sh
+    unuran-1.8.1/scripts/win32/example1.c
+    unuran-1.8.1/scripts/win32/example2.c
+    unuran-1.8.1/scripts/win32/make_dll_def.pl
+    unuran-1.8.1/src/Makefile.am
+    unuran-1.8.1/src/Makefile.in
+    unuran-1.8.1/src/README
+    unuran-1.8.1/src/distr/Makefile.am
+    unuran-1.8.1/src/distr/Makefile.in
+    unuran-1.8.1/src/distributions/Makefile.am
+    unuran-1.8.1/src/distributions/Makefile.in
+    unuran-1.8.1/src/distributions/README
+    unuran-1.8.1/src/methods/Makefile.am
+    unuran-1.8.1/src/methods/Makefile.in
+    unuran-1.8.1/src/parser/Makefile.am
+    unuran-1.8.1/src/parser/Makefile.in
+    unuran-1.8.1/src/parser/functparser_doc.dh
+    unuran-1.8.1/src/parser/make_stringparser.pl
+    unuran-1.8.1/src/parser/stringparser_doc.dh
+    unuran-1.8.1/src/specfunct/Makefile.am
+    unuran-1.8.1/src/specfunct/Makefile.in
+    unuran-1.8.1/src/tests/Makefile.am
+    unuran-1.8.1/src/tests/Makefile.in
+    unuran-1.8.1/src/uniform/Makefile.am
+    unuran-1.8.1/src/uniform/Makefile.in
+    unuran-1.8.1/src/uniform/fish.c
+    unuran-1.8.1/src/uniform/mrg31k3p.c
+    unuran-1.8.1/src/uniform/mstd.c
+    unuran-1.8.1/src/uniform/unuran_urng_gsl.h
+    unuran-1.8.1/src/uniform/unuran_urng_prng.h
+    unuran-1.8.1/src/uniform/unuran_urng_rngstreams.h
+    unuran-1.8.1/src/uniform/urng_builtin.c
+    unuran-1.8.1/src/uniform/urng_builtin.h
+    unuran-1.8.1/src/uniform/urng_fvoid.c
+    unuran-1.8.1/src/uniform/urng_fvoid.h
+    unuran-1.8.1/src/uniform/urng_gsl.c
+    unuran-1.8.1/src/uniform/urng_gsl.h
+    unuran-1.8.1/src/uniform/urng_gslqrng.c
+    unuran-1.8.1/src/uniform/urng_gslqrng.h
+    unuran-1.8.1/src/uniform/urng_prng.c
+    unuran-1.8.1/src/uniform/urng_prng.h
+    unuran-1.8.1/src/uniform/urng_randomshift.c
+    unuran-1.8.1/src/uniform/urng_randomshift.h
+    unuran-1.8.1/src/uniform/urng_rngstreams.c
+    unuran-1.8.1/src/uniform/urng_rngstreams.h
+    unuran-1.8.1/src/unuran.h.in
+    unuran-1.8.1/src/urng/Makefile.am
+    unuran-1.8.1/src/urng/Makefile.in
+    unuran-1.8.1/src/utils/Makefile.am
+    unuran-1.8.1/src/utils/Makefile.in
+    unuran-1.8.1/tests/Makefile.am
+    unuran-1.8.1/tests/Makefile.in
+    unuran-1.8.1/tests/README.conf
+    unuran-1.8.1/tests/make_test_files.pl
+    unuran-1.8.1/tests/t_arou.c
```

Comment:
> let's discuss how you decided to remove some of the tests

These are UNU.RAN's unit/regression tests and not the `tests` submodule (if you are confused between the two :)). The `tests` directory with statistical tests in them is kept intact. I removed the unit tests because I was unaware of how to compile them (as they come with a lot of configuration files and code generation using Perl scripts) and whether we want to test them on SciPy.

Reply:
we have decided that we will include some of the tests from the `tests` directory manually to verify the correctness of the UNU.RAN implementation in SciPy and to remove the directory